alpine-nix-rails-nginx/etc/sv/rails

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+cd /opt/rails
+exec sh -c "$(sed -n 's/#.*$//;/^$/d;s/^web:\s*//gp' Procfile)"
```

Comment:
Yeah, this is probably better than evaluating it by hand. As much as I think using ruby this way is probably overkill, I think it's probably the simplest command I can get on the system without installing another dependency.